### PR TITLE
Improve the Azure documentation

### PR DESCRIPTION
**Pull request by jeanbisutti**
cc @brunobat

### CODE REVIEW

File: docs/modules/ROOT/pages/quarkus-opentelemetry-exporter-azure.adoc [hunk 1]
```diff
@@ -23,27 +19,29 @@ For Maven:
 </dependency>
 ----
 
-`quarkus-opentelemetry` extension already includes the default OTLP exporter. To deactivate that default exporter and only have this Azure extension exporting data, please set the following property in the `application.properties`:
```
Comment (jeanbisutti) [inline docs/modules/ROOT/pages/quarkus-opentelemetry-exporter-azure.adoc:26]: See https://github.com/Azure-Samples/java-native-telemetry/pull/2#discussion_r1532181463

File: docs/modules/ROOT/pages/quarkus-opentelemetry-exporter-azure.adoc [hunk 2]
```diff
@@ -23,27 +19,29 @@ For Maven:
...
 ----
 
-You need an Application Insights resource receiving the telemetry data. Go to the Azure portal, search for your Application Insights resource or create a new one. On the overview page of your resource, you will find a connection string [in the top right corner](https://learn.microsoft.com/azure/azure-monitor/app/sdk-connection-string).
+You also need an Application Insights resource receiving the telemetry data. Go to the Azure portal, search for your Application Insights resource or create a new one. On the overview page of your resource, you will find a connection string [in the top right corner](https://learn.microsoft.com/azure/azure-monitor/app/sdk-connection-string).
+
+Provide this connection string in the `application.properties` file:
 
 [source]
 ----
 quarkus.otel.azure.applicationinsights.connection.string=your_connection_string
```
Comment (jeanbisutti) [inline docs/modules/ROOT/pages/quarkus-opentelemetry-exporter-azure.adoc:35]: Is it supposed to work with an environmental variable?QUARKUS_OTEL_AZURE_APPLICATIONINSIGHTS.CONNECTION_STRING?

File: docs/modules/ROOT/pages/quarkus-opentelemetry-exporter-azure.adoc [hunk 3]
```diff
@@ -23,27 +19,29 @@ For Maven:
 </dependency>
 ----
 
-`quarkus-opentelemetry` extension already includes the default OTLP exporter. To deactivate that default exporter and only have this Azure extension exporting data, please set the following property in the `application.properties`:
-
-[source]
-----
-quarkus.opentelemetry.tracer.exporter.otlp.enabled=false
```
Comment (brunobat) [inline docs/modules/ROOT/pages/quarkus-opentelemetry-exporter-azure.adoc:30]: It's on by default

**Review by brunobat — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/quarkus-opentelemetry-exporter-azure.adoc [hunk 4]
```diff
@@ -23,26 +19,31 @@ For Maven:
...
 ----
 
-Enable the Azure export by adding following properties in the `application.properties`:
+* Or as an environment variable: `QUARKUS_OTEL_AZURE_APPLICATIONINSIGHTS.CONNECTION_STRING=your_connection_string`
+
+You can disable the Azure export in this way:
 
 [source]
 ----
 quarkus.opentelemetry.tracer.exporter.azure.enabled=true
```
Comment (brunobat) [inline docs/modules/ROOT/pages/quarkus-opentelemetry-exporter-azure.adoc:38]: To disable it should be 
`quarkus.opentelemetry.tracer.exporter.azure.enabled=false`

**Review by brunobat — APPROVED** [review]
Perfect. Thanks @jeanbisutti

**Pull request merged**